stories/90-Components/Organisms/Accordions.stories.js

```diff
@@ -0,0 +1,54 @@
+import VAccordion from '../../../src/components/VAccordion'
+
+export default {
+  title: 'Components/Organisms/Accordions'
+}
+
+export const Default = () => ({
+  components: {
+    VAccordion
+  },
+  template: `
+    <div>
+    <v-accordion class="u-space--bottom">
+      <template slot="header"><h2>Header 1</h2></template>
+      <template slot="content">
```

Comment:
i was reading up in the Vue manual to understand slots better, and i noticed that as of 2.6, `slot="header"` here is deprecated in favor of `v-slot="header"` or the shorthand `#header`. Same for `<template #content>`

https://vuejs.org/v2/guide/components-slots.html#Named-Slots

I know you might already know this and are just copying over some older code, and also it's just a test not production code, so no big deal. just something i noticed.

Reply:
oh good catch - yeah I was copying older code. I'll update to the new way!

Reply:
especially important to keep up to date because people here will be learning a lot of Vue from copying your code 😆